lapce-app/src/command.rs

```diff
@@ -498,6 +498,10 @@ pub enum LapceWorkbenchCommand {
     #[strum(serialize = "previous_error")]
     PreviousError,
 
+    #[strum(serialize = "panic")]
+    #[strum(message = "Panic Editor")]
+    Panic,
```

Comment:
Do we really want this command in the palette?

Reply:
I don't think there is anything wrong with that

Reply:
I find it odd to have a command to crash your editor.

Reply:
For basic users, that's true, but when developing the editor it sometimes might be helpful (like testing this change) I can remove it though if you want

Reply:
Yes please. For testing, you can just panic!() after the hook?